### PR TITLE
Smoke tests

### PR DESCRIPTION
**Pull request by balteravishay**
Adding docs about smoke testing.
updated to the new automated-testing template.

### CODE REVIEW

File: automated-testing/smoke-testing/readme.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+### ROI Tipping Point
+
+Smoke tests cover only the most critical application path, and should not be used to actually test the application's behaviour, keeping execution time and complexity to minimum. The tests can be formed of a subset of the application's integration or e2e tests, and they cover as much of the functionality with as little depth as required.
+
+The golden rule of a good smoke test is that it saves time on validating that the application is acceptable to a stage where better, more thorough testing will begin.
+
+### Applicable to
+
+- [ ] **Local dev 'desktop'** - *Example:* Applying manual smoke testing to verify that the application is OK.
```
Comment (TessFerrandez) [inline automated-testing/smoke-testing/readme.md:19]:
```suggestion
- [ ] **Local dev desktop'** - *Example:* Applying manual smoke testing to verify that the application is OK.
```

**Review by TessFerrandez — APPROVED** [review]
LGTM - just one nit

File: automated-testing/smoke-testing/readme.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+Smoke tests cover only the most critical application path, and should not be used to actually test the application's behaviour, keeping execution time and complexity to minimum. The tests can be formed of a subset of the application's integration or e2e tests, and they cover as much of the functionality with as little depth as required.
+
+The golden rule of a good smoke test is that it saves time on validating that the application is acceptable to a stage where better, more thorough testing will begin.
+
+### Applicable to
+
+- [ ] **Local dev desktop'** - *Example:* Applying manual smoke testing to verify that the application is OK.
+- [ ] **Build pipelines** - *Example:* Running a small set of the integration test suite before running the full coverage of tests, which may take a long time.
+- [ ] **Non-production and Production deployments** - *Example:* Running a curl command to the product's API and asserting the response is 200 before running load test which consume resources.
+- [ ] **PR Validation** - *Example:* - Deploying the application chart to a test namespace and validating the release is successful and no immediate regressions are merged.
```
Comment (RichCzyzewski) [inline automated-testing/smoke-testing/readme.md:22]: The applicable ones should have a check:
- [x] Applicable
- [ ] Not applicable

**Review by RichCzyzewski — CHANGES_REQUESTED** [review]
Great write up. One minor issue though, we probably should check the check boxes on the "applicable to" section.

**Review by RichCzyzewski — APPROVED** [review]

**Pull request merged**